Standards/scs-0001-v2-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,309 @@
+
+The file name of an SCS document is formed using the following pattern:
+`scs-XXXX-vN-T.md`, where
+`XXXX` is replaced with the zero-padded document number,
+`N` is replaced with the major version of the document, and
+`T` is replaced with the slugified title.
+For a document with the number 190, with a major version number 2 and a slugified title `flavor-naming`,
+the resulting file name would be `scs-0190-v2-flavor-naming.md`.
+
+The second digit in `XXXX` describes the track where the document belongs:
+
+| Track  | Number |
+|--------|--------|
+| Global | 0      |
+| IaaS   | 1      |
+| KaaS   | 2      |
+| IAM    | 3      |
+| Ops    | 4      |
+
+In addition to the number, each document has the following metadata,
+embedded in the markdown header.
+
+| Field name      | Requirement                                                                | Description                                                                           |
+|-----------------|----------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| `type`          | REQUIRED                                                                   | one of `Procedural`, `Standard`, or `Decision Record`                                 |
+| `status`        | REQUIRED                                                                   | one of `Proposal`, `Draft`, `Stable`, `Deprecated`, or `Rejected`                     |
+| `track`         | REQUIRED                                                                   | one of `Global`, `IaaS`, `KaaS`, `IAM`, `Ops`                                         |
+| `obsoleted_at`  | REQUIRED if `status` is `Deprecated`                                       | ISO formatted date indicating the date after which the deprecation is in effect       |
+| `stabilized_at` | REQUIRED if `status` was ever `Stable`                                     | ISO formatted date indicating the date after which the document was considered stable |
+| `rejected_at`   | REQUIRED if `status` is `Rejected`                                         | ISO formatted date indicating the date on which the document was rejected             |
+| `replaced_by`   | RECOMMENDED if `status` is `Deprecated` or `Rejected`, FORBIDDEN otherwise | List of documents which replace this document.                                        |
+| `replaces`      | OPTIONAL                                                                   | List of documents which this document replaces.                                       |
+
+### Sections
+
+#### Standard
+
+Each Standard document MUST have the following sections:
+
+- An _Introduction_ providing context on the document and linking to other relevant materials.
+- A _Motivation_ section which details why this document or the thing it describes is necessary.
+- A _Standard_ section containing the actual standardization decision.
```

Comment:
I'm still not convinced of this. A document that is a _Standard_ must have a section named _Standard_? No, that doesn't seem right. From my POV, there are certain sections that more or less suggest themselves, such as Introduction, Motivation, and the like, but the actual content would have sections that are specific to the topic at hand. You don't find scientific articles that have a section "article (the actual one)" or "content" or similar; no, the content proper is in sections that are specific to the topic at hand and the style of the article.

Reply:
I agree. I prefer to split the actual content in REQUIRED, RECOMMENDED, OPTIONAL requirements and dictate to have at least REQUIRED or RECOMMENDED sections.

Reply:
I'm not sure about these sections either, if @mbuechse isn't happy with the `Standard` section. I think the text flow of the document is considered more important here, so I wouldn't force these sections on the document either, also because they split some logically connected sections up.